recipes/gh/bld.bat

```diff
@@ -0,0 +1,13 @@
+:: Turn work folder into GOPATH
+set GOPATH=%SRC_DR%
+set PATH=%GOPATH%\bin:%PATH%
+
+:: Change to directory with main.go
+cd cmd\gh
+
+:: Build
+go build -v -o %PKG_NAME%.exe .
+
+:: Install Binary into %PREFIX%\bin
+mkdir -p %PREFIX%\bin
+mv %PKG_NAME% %PREFIX%\bin\%PKG_NAME%
```

Comment:
This should have `if errorlevel 1 exit 1` after every line.  Example at https://github.com/conda-forge/make-feedstock/blob/master/recipe/bld.bat

Reply:
changes made